install.rst

```diff
@@ -99,19 +99,7 @@ You can get the `official PostgREST Docker image <https://hub.docker.com/r/postg
 
   docker pull postgrest/postgrest
 
-The image consults an internal ``/etc/postgrest.conf`` file. To customize this file you can either mount a replacement configuration file into the container, or use environment variables. The environment variables will be interpolated into the default config file.
```

Comment:
This breaking change should be listed in Upcoming.

Reply:
Btw, why the config file inside docker don't work anymore?

Reply:
I see that the Nix docker script did include the [config file](https://github.com/PostgREST/postgrest/pull/1522/files#diff-efc6bf6ccfbaf2c72cc63bcb3b11e5871273bbd23a7bf63e5e790f16aca835edR18) but at some point it was removed([current](https://github.com/PostgREST/postgrest/blob/main/nix/tools/docker/default.nix)).

Reply:
Ah, it was removed in https://github.com/PostgREST/postgrest/pull/1691/. This should be noted in the CHANGELOG.